### ISSUE TITLE
WebSaver not working on OS X El Capitan

### ISSUE THREAD

**Issue by mdalikazi** [brockgr/websaver#20]
Any upcoming updates?

![screen shot 2015-12-22 at 3 55 39 pm](https://cloud.githubusercontent.com/assets/12589127/11948133/7cb9d342-a8c4-11e5-9807-fb1e052b751b.png)

**Comment by jmpace** [comment]
Yes, I would love to see this working in El Cap. I use it as a "suggested use" with my website Colorsinc.co, not to mention this screen saver is just awesome and useful all over the place.

**Comment by brianclaveau-coresecure** [comment]
Any chance of an update?

**Comment by TomasHubelbauer** [comment]
Folks, please check again, I am a new user and I just noticed that the project has been recently updated and I managed to compile it successfully with no hiccups on macOS 13 Ventura. I see these comments are a couple of years old but if you still have an interest, chances are this screen saver will now work for you!
